### DISCUSSION TITLE
編集中の編集リクエストの扱い方。

### DISCUSSION THREAD

**Discussion by kaizen-nagoya** [increments/qiita-discussions#153] [要望・アイデア・不具合]
文書構造を変更している最中に、文書構造についての編集リクエストが来ていたらしく、自分の変更を保存した後で気がつくと、どうしたらいいかわからないことがありました。

それ以来、文書構造の編集リクエストは送らないでくださいとお願いするようにしていました。

それぞれの変更を分岐して、分岐したものを比較して、どこを採用するか決められるような仕組みになっていると嬉しいかも。

すでにそういう仕組みがあって、使い方がわかっていないだけだったらごめんなさい。

**Comment by kskwtnk** [comment]
@kaizen-nagoya 
ご意見ありがとうございます！

まず、記載いただいた状況において差分があまりにも分かりづらい事実そのものがこれまで把握できていませんでした
申し訳ないです 🙏 

そして、現状どのようにすると分かりやすく変更差分を示せるかの目処が立っていません
運営としては編集リクエストのやり取りは活発になって欲しいため、改修のためにこれから議論を進めていきます 🙇
